### ISSUE TITLE
Multiple dex files define Lwseemann/media/Metadata

### ISSUE THREAD

**Issue by vo1kov** [wseemann/FFmpegMediaMetadataRetriever#74]
I try to use both FFmpegMediaPlayer and FFmpegMediaMetadataRetriever in one android application.
And have Multiple dex files define Lwseemann/media/Metadata  error on gradle build.

**Comment by fxhereng** [comment]
@vo1kov Were you able to solve this? 

I'm facing the same issue, I have both libs in my project and having this error : 
```
Error:Execution failed for task ':app:transformClassesWithJarMergingForDebug'.
> com.android.build.transform.api.TransformException: java.util.zip.ZipException: duplicate entry: wseemann/media/Metadata.class
```

**Comment by wseemann** [comment]
This is fixed in the latest version (v1.0.5).

**Comment by fxhereng** [comment]
Thanks, it works fine now.
